src/Kiota.Builder/Refiners/PythonRefiner.cs

```diff
@@ -172,8 +172,6 @@ private static void CorrectPropertyType(CodeProperty currentProperty)
             currentProperty.Type.Name = "List[RequestOption]";
         else if (currentProperty.IsOfKind(CodePropertyKind.Headers))
             currentProperty.Type.Name = "Dict[str, Union[str, List[str]]]";
-        else if (currentProperty.IsOfKind(CodePropertyKind.QueryParameters))
```

Comment:
@samwelkanda I've made a change in this PR were the code type name maps directly to the type definition name when provided. This avoid strings duplication and having to do the same work multiple time. The only side effect I've noticed is here. I've made a modification in the code property writer. Could you confirm I didn't miss anything please?